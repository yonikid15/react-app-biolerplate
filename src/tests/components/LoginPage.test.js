import React from 'react';
import { shallow } from 'enzyme';
import { LoginPage } from '../../components/LoginPage';

test('should render the correct snapshot', () => {
    const wrapper = shallow(<LoginPage />);
    expect(wrapper).toMatchSnapshot();
});

test('should call startLogin on button click', () => {
    const onStartLoginSpy = jest.fn();
    const wrapper = shallow(<LoginPage startLogin={onStartLoginSpy}/>);
    wrapper.find('button').simulate('click');
    expect(onStartLoginSpy).toHaveBeenCalled();
});