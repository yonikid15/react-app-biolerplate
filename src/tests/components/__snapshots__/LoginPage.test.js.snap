// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the correct snapshot 1`] = `
<div
  className="wrapper"
>
  <h1
    className="title"
  >
    Boilerplate
  </h1>
  <p>
    Tag line for app.
  </p>
  <button
    className="btn btn-teal btn-pos"
  >
    Login
  </button>
</div>
`;
